### ISSUE TITLE
`latest` is no longer adding labels

### ISSUE THREAD

**Issue by benjivesterby** [BetaHuhn/repo-file-sync-action#146]
**🐞 Describe the bug**

I have this setup for a number of repositories. I was using `latest` and I've also tried `@v1`

This was working last time I added changes. 

**📚 To Reproduce**

Run workflow with labels specified

**💡 Expected behavior**

The label should be added to the PR

**🖼️ Screenshots**

Here is one of the PRs from the last couple runs with `latest` and `@v1`: [PR](https://github.com/devnw/bridgekeeper/pull/22)

I tried running multiple times and the label is not added.

**⚙️ Environment**
- Action version: latest

**Comment by benjivesterby** [comment]
I've verified that this is not an issue in `v1.15.5` Also, the assignee is not being set in the latest. 
Here's a PR with v1.15.5 working: https://github.com/devnw/bridgekeeper/pull/24

**Comment by benjivesterby** [comment]
I removed the link to the workflow since I updated it to use `v1.15.5`.

**Comment by BetaHuhn** [comment]
If it does work in `v1.15.5`, then I suspect that https://github.com/BetaHuhn/repo-file-sync-action/commit/80e77fd377b039131927d4983859ec47fa23b806 broke the labels and assignees.

Will take a closer look.

**Comment by BetaHuhn** [comment]
I might have found the issue already, could you please try the fix on the [develop](https://github.com/BetaHuhn/repo-file-sync-action/tree/develop) branch:

```yml
uses: BetaHuhn/repo-file-sync-action@develop
```

**Comment by benjivesterby** [comment]
@BetaHuhn Yup, looks like that fixed it

**Comment by BetaHuhn** [comment]
Great, will merge it into master and release the fix.

**Comment by BetaHuhn** [comment]
This issue is resolved in version [v1.16.1](https://github.com/BetaHuhn/repo-file-sync-action/releases/tag/v1.16.1) :tada:

To use the latest version, use the Action with the `v1` or `latest` tag:

```yml
uses: BetaHuhn/repo-file-sync-action@v1
```

Also thanks for supporting me via GitHub Sponsors @benjivesterby!
